integration-libs/cds/package.json

```diff
@@ -28,6 +28,7 @@
     "@angular/core": "^10.1.0",
     "@angular/router": "^10.1.0",
     "@ngrx/store": "^10.0.0",
+    "@spartacus/checkout": "3.0.0",
```

Comment:
the version is 3.0.0 or 3.2.0?

Reply:
Latest release is 3.3.0.  I'll align on that as a baseline.  In the end, checkout lib should be first released as 4.0

Reply:
Actually I'll use `3.2.0-next.2`. This version on develop is intentional.